packages/react/src/ActionMenu/ActionMenu.tsx

```diff
@@ -392,9 +392,9 @@ const ActionMenuButton = forwardRef<HTMLButtonElement, ActionMenuButtonProps>(
             variant={variant}
             hasArrow={false}
             aria-haspopup="true"
-            aria-label={`${menuOpen ? 'Close' : 'Open'} menu`}
+            aria-label="Action menu"
```

Comment:
While "Action menu" is our internal UI component name, it shouldn't be announced on our marketing pages this way... Can we simply call this a "Menu"?